Commandant/Errors.swift

```diff
@@ -39,7 +39,11 @@ internal func informativeUsageError<T>(keyValueExample: String, option: Option<T
 		description += "]"
 	}
 
-	description += "\n\t\(option.usage)"
+	description += option.usage.componentsSeparatedByString("\n")
```

Comment:
It is, but the advantage is that it takes care of _all_ “line terminators.”

I guess since this is provided programmatically, though, we needn't worry about it.